model/src/id.rs

```diff
@@ -187,6 +204,23 @@ impl From<u64> for IntegrationId {
     }
 }
 
+#[derive(
+    Clone, Copy, Debug, Default, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize,
+)]
+pub struct InteractionId(#[serde(with = "string")] pub u64);
```

Comment:
I'm wondering if this and `CommandId` should be somewhere else, there's precedent with [OAuth IDs](https://docs.rs/twilight-model/0.4.0/twilight_model/oauth/id/index.html).

Reply:
Since slash commands are accessible over both the gateway and via a webhook client, I'm not quite sure.